exercises/raindrops/metadata.toml

```diff
@@ -2,3 +2,5 @@ title = "Raindrops"
 blurb = "Convert a number to a string, the content of which depends on the number's factors."
 source = "A variation on FizzBuzz, a famous technical interview question that is intended to weed out potential candidates. That question is itself derived from Fizz Buzz, a popular children's game for teaching division."
 source_url = "https://en.wikipedia.org/wiki/Fizz_buzz"
+
+deep_dive_blurb = "Explore 14 different ways to solve Raindrops including functional pipelines, matrix multiplication, a precalculated table-based solutions using a macro, an Object Oriented approach, some impressive use of both Python and TCL, and an enterprise edition. Kick back and enjoy 45mins of learning with Jeremy and Erik."
```

Comment:
"Enterprise edition" isn't clear here.

Reply:
```suggestion
deep_dive_blurb = "Explore 14 different ways to solve Raindrops including functional pipelines, matrix multiplication, a precalculated table-based solutions using a macro, an Object Oriented approach, some impressive use of both Python and Tcl, and an enterprise edition. Kick back and enjoy 45mins of learning with Jeremy and Erik."
```

Reply:
> "Enterprise edition" isn't clear here.

It is when you watch the video 😉 

On a serious note, I'm not sure how else to say it though.